sys/android-architecture.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```

Comment:
Hm, @radare has done in other commits, so I guess this is safe to merge or discard now?